### PR TITLE
NodeJS Rewrite :)

### PR DESCRIPTION
**Pull request by Looskie**
## Description

NodeJS rewrite of entire website

Fixes everything 🤤 

## Type of change

Please tick the relevant option by putting an X inside the bracket

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document

### CODE REVIEW

**Comment by Looskie** [comment]
i'd still like to work on the view all blogs before we merge pwetty pwease 👉 👈

File: routes/indexRouter.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+routes.use('/docs', docsRouter);
+
+// Login/register stuff with account stuff
+
+routes.use('/login', notAuth, loginRouter);
+routes.use('/register', auth, registerRouter);
+routes.use('/dashboard', auth, dashboardRouter)
+
+routes.get('/logout', (req, res) => {
```
Comment (Looskie) [inline routes/indexRouter.js:55]: does this work?

Comment (Looskie) [reply routes/indexRouter.js:55]: oh wow it does! isnt that cool?!

Comment (HexaOneOfficial) [reply routes/indexRouter.js:55]: I don't know much about js but thats a lot of code updated

Comment (Looskie) [reply routes/indexRouter.js:55]: > I don't know much about js but thats a lot of code updated

yeahhh I practically rewrote the entire website in nodejs :)

**Comment by nt4f04uNd** [comment]
does this pr adress https://github.com/dahliaOS/website/issues/25?

**Comment by SincerelyFaust** [comment]
It indeed does.

**Comment by HexaOneOfficial** [comment]
Could Documentation als be updated as its a really old version

**Comment by Looskie** [comment]
> Could Documentation als be updated as its a really old version

gotcha, will update soon! thank you

**Comment by bleonard252** [comment]
I'll add the New Docs to this, once it's merged into a submodule-able repository.

**Pull request merged**